test/unit/rounding/fixed_point/elastic_integer/elastic_fixed_point.cpp

```diff
@@ -112,4 +112,81 @@ namespace {
                 -10),
                 "");
     }
+
+    namespace efxp_round_test{
+        using T0 = cnl::elastic_fixed_point<24, -20>;
+        using T1 = cnl::elastic_fixed_point<16, -8>;
+        using T2 = cnl::elastic_fixed_point<8, -4>;
+        using T3 = cnl::elastic_fixed_point<5, -1>;
+    }
+
+    namespace elastic_fixed_point_implicit_conversions {
+        using efxp_round_test::T1;
```

Comment:
Why put these types in a namespace and then pull them into another namespace? The whole thing is already in an anonymous namespace. It all serves to make it harder work for the reader to understand what's going on in the file.

Reply:
I was lazy to name the types to something specific. I just wanted to call them T0...T3 simply for the reason for me to not to rewrite all the code for different type tests (nearest, neg_inf, tie_to_pos_inf). All I needed to do was to change the results for some specific testcases. Initially when the T0,...T3 types were in anonymous namespace, the different type definitions using the same name in different files conflicted, thus I needed to put them inside some specific namespace. Then again to save some text then the pull in to actual test namespace.